packages/fds-components/src/style/buttons/buttonGroup.css

```diff
@@ -0,0 +1,70 @@
+:root {
+  --button-group-timeout: 250ms;
+}
+
+.btn-group {
+  display: inline-flex;
+}
+
+.btn-group > .group-btn {
```

Comment:
We probably don't need these direct child combinators for buttons that are ancestors of `btn-group`.

If someone is deeply nesting buttons inside buttons, they're doing something wrong. We may also be able to guarantee ancestry if we want to go with a `label` prop on `GroupButton` instead of using `children`.